### PR TITLE
chore: Rename eth to evm

### PR DESCRIPTION
**Pull request by SweeXordious**
Renames `Eth` to `EVM` accross the repo to reflect the fact that the QGB can be deployed over any EVM chain, instead of only Ethereum.

Contributes to: https://github.com/celestiaorg/cosmos-sdk/issues/136

### CODE REVIEW

**Review by liamsi — APPROVED** [review]
I do not know if this has any implications in terms of compatibility but the renaming looks good to me.

**Comment by SweeXordious** [comment]
> I do not know if this has any implications in terms of compatibility but the renaming looks good to me.

You mean on Celestia-app side? I will open a PR afterwards to accommodate this change.

If you mean target EVM chains, any EVM chain can deploy the QGB contract and all it will need is a relayer to submit the signatures to it. Nothing special for Ethereum. John and Evan can correct me

File: x/genutil/client/cli/gentx.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ $ %s gentx my-key-name 1000000stake --home=/path/to/home/dir --keyring-backend=o
     --security-contact="..." \
     --website="..." \
     --orchestrator-address="..." \
-    --ethereum-address="..."
+    --evm-address="..."
```
[the remark this answers is not in the document]
Comment (SweeXordious) [reply x/genutil/client/cli/gentx.go:56]: After discussing this, the QGB can be used for any chain that supports the EVM. Thus, the naming is referring to any EVM compatible key, which includes, but not only, Ethereum.

File: x/staking/types/validator.go [hunk 2]
```diff
@@ -418,7 +419,8 @@ func (v Validator) RemoveTokens(tokens sdk.Int) Validator {
 
 // RemoveDelShares removes delegator shares from a validator.
 // NOTE: because token fractions are left in the valiadator,
-//       the exchange rate of future shares of this validator can increase.
+//
```
[the remark this answers is not in the document]
Comment (rootulp) [reply x/staking/types/validator.go:422]: Let's leave it because not worth changing.

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**